app/src/main/res/values/strings.xml

```diff
@@ -952,6 +952,34 @@
     <string name="hko_warning_text_fire_red_description" translatable="false">The Fire Danger Warning is Red and the fire risk is Extreme.</string>
+    <!-- Weather texts for MGM -->
+    <string name="mgm_weather_text_clear" translatable="false">@string/common_weather_text_clear_sky</string>
+    <string name="mgm_weather_text_slightly_cloudy" translatable="false">@string/common_weather_text_mainly_clear</string>
+    <string name="mgm_weather_text_partly_cloudy" translatable="false">@string/common_weather_text_partly_cloudy</string>
+    <string name="mgm_weather_text_mostly_cloudy" translatable="false">@string/common_weather_text_cloudy</string>
+    <string name="mgm_weather_text_rain_light" translatable="false">@string/common_weather_text_rain_light</string>
+    <string name="mgm_weather_text_rain" translatable="false">@string/common_weather_text_rain</string>
+    <string name="mgm_weather_text_rain_heavy" translatable="false">@string/common_weather_text_rain_heavy</string>
+    <string name="mgm_weather_text_sleet" translatable="false">@string/weather_kind_sleet</string>
+    <string name="mgm_weather_text_snow_light" translatable="false">@string/common_weather_text_snow_light</string>
+    <string name="mgm_weather_text_snow" translatable="false">@string/common_weather_text_snow</string>
+    <string name="mgm_weather_text_snow_heavy" translatable="false">@string/common_weather_text_snow_heavy</string>
+    <string name="mgm_weather_text_rain_showers_light" translatable="false">@string/common_weather_text_rain_showers_light</string>
+    <string name="mgm_weather_text_rain_showers" translatable="false">@string/common_weather_text_rain_showers</string>
+    <string name="mgm_weather_text_rain_showers_heavy" translatable="false">@string/common_weather_text_rain_showers_heavy</string>
+    <string name="mgm_weather_text_rain_showers_local">Local rain showers</string>
+    <string name="mgm_weather_text_hail" translatable="false">@string/weather_kind_hail</string>
+    <string name="mgm_weather_text_thunderstorm" translatable="false">@string/weather_kind_thunderstorm</string>
+    <string name="mgm_weather_text_thunderstorm_heavy">Heavy thunderstorm</string>
+    <string name="mgm_weather_text_fog" translatable="false">@string/common_weather_text_fog</string>
+    <string name="mgm_weather_text_mist" translatable="false">@string/common_weather_text_mist</string>
+    <string name="mgm_weather_text_smoke" translatable="false">@string/common_weather_text_smoke</string>
+    <string name="mgm_weather_text_dust_or_sand_storm" translatable="false">@string/common_weather_text_sand_storm</string>
+    <string name="mgm_weather_text_wind" translatable="false">@string/weather_kind_wind</string>
+    <string name="mgm_weather_text_strong_southerly_wind">Strong southerly wind</string>
+    <string name="mgm_weather_text_strong_northerly_wind">Strong northerly wind</string>
```

Comment:
Missing `translatable="false"`

Reply:
If we don't make them non-translatable, can we point them to another string instead? Like `wind_strength_X`. The direction is already shown in the app.

Reply:
No definition found. I think I'll just point to `weather_kind_wind` for both.